script/ProductData.json

```diff
@@ -0,0 +1,11 @@
+[{"title":"Pahat pojat","description":"Sed ante. Vivamus tortor. Duis mattis egestas metus.\n\nAenean fermentum. Donec ut mauris eget massa tempor convallis. Nulla neque libero, convallis eget, eleifend luctus, ultricies eu, nibh.","price":53,"inventory":70,"imageUrl":"http://dummyimage.com/226x180.png/5fa2dd/ffffff"},
+{"title":"Family Guy Presents: Blue Harvest","description":"Nulla ut erat id mauris vulputate elementum. Nullam varius. Nulla facilisi.\n\nCras non velit nec nisi vulputate nonummy. Maecenas tincidunt lacus at velit. Vivamus vel nulla eget eros elementum pellentesque.\n\nQuisque porta volutpat erat. Quisque erat eros, viverra eget, congue eget, semper rutrum, nulla. Nunc purus.","price":69,"inventory":56,"imageUrl":"http://dummyimage.com/207x149.bmp/5fa2dd/ffffff"},
+{"title":"Peck on the Cheek, A (Kannathil Muthamittal)","description":"Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Vivamus vestibulum sagittis sapien. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus.","price":91,"inventory":47,"imageUrl":"http://dummyimage.com/145x231.png/5fa2dd/ffffff"},
+{"title":"Princess Protection Program","description":"Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Vivamus vestibulum sagittis sapien. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus.","price":47,"inventory":47,"imageUrl":"http://dummyimage.com/116x128.png/5fa2dd/ffffff"},
+{"title":"Hulk","description":"Phasellus in felis. Donec semper sapien a libero. Nam dui.","price":50,"inventory":98,"imageUrl":"http://dummyimage.com/221x235.png/dddddd/000000"},
```

Comment:
Can we add one or two for the seedfile where the price has some cents (e.g. 40.34).